templates/base.html.twig

```diff
@@ -1,19 +1,33 @@
+{# This is the base that all other views will extend #}
+
 <!DOCTYPE html>
 <html>
 <head>
     <meta charset="UTF-8">
     <link rel="icon" type="image/png" href="{{ asset('build/images/favicon.png') }}" />
-    <title>{% block title %}Welcome!{% endblock %}</title>
+    <title>{% block title %}Mobilean{% endblock %}</title>
     {% block stylesheets %}
         {{ encore_entry_link_tags('app') }}
     {% endblock %}
 </head>
 <body>
+    <header>
+    {% block navbar %}
+    {% endblock %}
+    </header>
 
-{% block body %}{% endblock %}
+    <main>
+    {% block body %}
+        <h1>{{ pageTitle }}</h1>
```

Comment:
You should probably avoid the default content of a block if you don't know the content. The pageTitle twig variable could be undefined